### PR TITLE
add ka-GE locale.

### PR DESCRIPTION
**Pull request by overflowz**
ka-GE locale, if needed of course.

### CODE REVIEW

File: src/i18n/ge.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+(function(jsGrid) {
+
+    jsGrid.locales.ge = {
```
Comment (tabalinas) [inline src/i18n/ge.js]: I believe it should be `jsGrid.locales.ka = `?

**Comment by tabalinas** [comment]
Thank you very much for the PR! 
Could you please address one comment? Thank you!

File: src/i18n/ge.js (hunk 1 above)
Comment (overflowz) [inline src/i18n/ge.js]: whatever suits best.
ka = Kartuli (on Georgian means 'Georgian')
GE = Georgian (on English).

**Comment by overflowz** [comment]
I'm sorry, I don't quite get what you meant by 'address one comment'. If it's github specific stuff, I'm kinda new here (but account is old).

File: src/i18n/ge.js (hunk 1 above)
Comment (tabalinas) [inline src/i18n/ge.js]: If we look at standard ISO 639-1 locale names https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes we see that it's `ka`. So I'd rather go with the standard name, if you don't mind. Thank you!

Comment (overflowz) [inline src/i18n/ge.js]: Sure!

**Pull request merged**